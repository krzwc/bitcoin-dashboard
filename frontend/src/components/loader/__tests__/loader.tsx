import React from 'react';
import Loader from 'components/loader';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Loader', () => {
    test('snapshot test', () => {
        const wrapper = shallow(<Loader />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
