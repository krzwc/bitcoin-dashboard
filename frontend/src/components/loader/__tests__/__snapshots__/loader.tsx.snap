// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader snapshot test 1`] = `
<div
  className="loader"
>
  <div
    className="ball"
  />
  <div
    className="ball"
  />
  <div
    className="ball"
  />
</div>
`;
